docs/en/observability/tab-widgets/add-apm-integration/content.asciidoc

```diff
@@ -41,25 +30,16 @@ image::./images/kibana-fleet-integrations-apm-overview.png[{fleet} showing APM i
 
 . On the **Add Elastic APM integration** page,
 define the host and port where APM Server will listen.
+Make a note of this value--you'll need it later.
 
-. Under **Apply to agent policy**, select the default policy.
-Click **Save and continue**.
-+
-In this guide, you add integrations to the default policy created by
-{kib}. After you learn the basics, you can create your own policies and assign
-them to agents.
-+
-[role="screenshot"]
-image::./images/add-integration-apm.png[{fleet} Add APM integration page]
+. Click **Save and continue**.
+This step takes a minute or two to complete. When it's done,
+you'll have an agent policy that contains an APM integration policy for the configuration you just specified.
 
-// lint disable apm-1
-. To see the updated policy, click the *Default policy* link.
-+
-The newly added APM integration should appear under **{fleet}** > **Agent policies** > **Default policy**,
-along with the default System integration.
+. To view the new policy, click **Agent policy 1**.
 +
 [role="screenshot"]
-image::./images/kibana-fleet-policies-default-with-apm.png[{fleet} showing default agent policy with apm-1 data source]
+image::./images/apm-agent-policy-1.png[{fleet} showing apm policy]
 +
-All {agent}s that use this policy will collect APM data from your instrumented services.
+Any {agent}s assigned to this policy will collect APM data from your instrumented services.
```

Comment:
Would you mind changing this in the logs/metrics quick start, too, so it's consistent? I think it's better because it matches what the user does in the UI.

Reply:
Sure thing.